src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -512,7 +512,8 @@ static void AddCtorToTypeDeclaration(CodeTypeDeclaration typeDeclaration, Method
 
         static void AddMethodToTypeDeclaration(CodeTypeDeclaration typeDeclaration, MethodDefinition member, HashSet<string> excludeAttributes)
         {
-            if (member.IsAssembly || member.IsPrivate || member.IsSpecialName)
+            var isOperator = member.Name.StartsWith("op_");
```

Comment:
Could we inline this so that it is only evaluated with IsSpecialName?

Reply:
Definitely.